src/js/entities-service/entities/patients.js

```diff
@@ -14,6 +14,9 @@ const _Model = BaseModel.extend({
   getFields() {
     return Radio.request('entities', 'patientFields:collection', this.get('_patient_fields'));
   },
+  getEngagement() {
```

Comment:
probably won't need to do `|| {}` anywhere so likely won't need this `get`.. I think it's likely we'll only need to get the data in the template itself